docs/user-tutorial/data-diagnosis.md

```diff
@@ -0,0 +1,134 @@
+---
+id: data-diagnosis
+---
+
+# Data Diagnosis
+
+## Introduction
+
+This tool is to validate and diagnose the SuperBench benchmarking results for hundreds or thousands of servers automatically as to whether they meet the hardware performance/quality standard according to the baseline of metrics in the benchmarks,  and finally provides a list of failed servers with detailed information.
+
+## Input
+
+The input mainly includes 2 files:
+
+ - **raw data**: jsonl file including multi nodes' results automatically generated by SuperBench runner.
+
+    `Tips: the file can be found as ${output-dir}/results-summary.jsonl after each successfully run.`
+
+ - **baseline**: yaml format file including the rules and criteria for the metrics in the benchmarks used for diagnosis.
```

Comment:
It uses YAML format and includes rules and criteria for each metrics for diagnosis.